pages/doc.js

```diff
@@ -144,6 +169,23 @@ export default class Documentation extends Component {
       })
   }
 
+  loadFile = ({ section, subsection, file, parseHeadings }) => {
+    this.setState({ load: true })
+    let sect = sidebar[section]
+    let subsect = sect.files[subsection]
+    let subfolder = subsect && subsect.folder
+    let folderpath = file.folder || subfolder || sect.folder
+    let filepath = file.indexFile || file.files || file
```

Comment:
`file.files[0]`?

all these logic ^^ is still very complicated

can we make it always accept file as a file, not as a section or something else?

Reply:
we can if we make the sidebar.json structure the same for any type of element